Platform/ApollolakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.h

```diff
@@ -69,6 +69,11 @@
 #include <Library/BootOptionLib.h>
 #include <ConfigDataCommonStruct.h>
 #include <PsdLib.h>
+#include <Guid/SmmInformationGuid.h>
+
+#define  VBT_OFFSET                   36
+#define  TSEG                         0xb8    // TSEG base
```

Comment:
Should this be in silicon package instead ?

Reply:
sure. will put them into silicon header file